lib/stdlib/test/ets_SUITE.erl

```diff
@@ -9061,6 +9064,17 @@ ets_apply(F, Args, Opts) ->
 ets_format_args(Args) ->
     lists:join(", ", [io_lib:format("~p", [A]) || A <- Args]).
 
+bound_maps(_Config) ->
+    T = ets:new('__bound_maps__', [ordered_set, public]),
```

Comment:
Minor nit, if you ever plan to parallelize these test cases, generating a unique atom can be done by concatenating test function name and unique_integer.